client/modules/Booking/Booking.js

```diff
@@ -181,32 +185,39 @@ export class Booking extends Component {
           .asHours();
     this.tmpCost=
       hours*this.state.selectedCar.vehicleType.hourlyRate;
+    if(discount.multiplier)
```

Comment:
its better to wrap the body of if statements in { } even if it is a single statement. it removes ambiguity, and also prevents unintended bugs like you add an extra statement for the conditional path but forget to wrap the now multi-statement block